nym-vpn/ui/src/state/provider.tsx

```diff
@@ -41,6 +42,19 @@ export function MainStateProvider({ children }: Props) {
       return await invoke<Country>('get_default_node_location');
     };
 
+    getVersion()
+      .then((version) =>
+        dispatch({
+          type: 'set-version',
+          version,
+        }),
+      )
+      .catch((e: CmdError) => {
```

Comment:
here error is not of type `CmdError` as `getVersion` comes from tauri API
So you want it as `e: any`, and just logging the whole error object
```
        console.warn(`command [set-version] returned an error: ${e}`);
```